### ISSUE TITLE
`canUserEditRepo` is broken?

### ISSUE THREAD

**Issue by fregante** [fregante/github-url-detection#61]
https://github.com/sindresorhus/refined-github/pull/3783 added this line

```js
// Hide empty meta if it’s not editable by the current user
// TODO: once it's fixed, use https://github.com/fregante/github-url-detection/blob/4840d85d31d59bfe71e884ef42c482fbfff2d955/index.ts#L490
if (!select.exists('.repository-content [aria-label="Edit repository metadata"]')) {
```

Implying that `canUserEditRepo` is broken. Needs to be checked on multiple pages to make sure it works and where it doesn't.

cc @cheap-glitch since you wrote that comment you might have more details.

**Comment by cheap-glitch** [comment]
Actually https://github.com/sindresorhus/refined-github/pull/3974 removes this comment and uses `canUserEditRepo()` again (https://github.com/sindresorhus/refined-github/pull/3974/files#diff-b20811bb8c481b5e2b61bea4a347072bb0e5283e4afacfa6d3240adf4a841a99L24-R74), since this was fixed in #55 which was released in https://github.com/fregante/github-url-detection/releases/tag/v4.5.2.

**Comment by fregante** [comment]
Perfect, thanks!
